local/src/core/ProxyVerifier.cc

```diff
@@ -1927,64 +1993,114 @@ EqualityCheck::EqualityCheck(swoc::TextView name, swoc::TextView value)
 {
   _name = name;
   _value = value;
+  _is_field = true;
+}
+
+EqualityCheck::EqualityCheck(YamlUrlPart url_part, swoc::TextView value)
+{
+  _name = URL_PART_TO_NAME[static_cast<std::size_t>(url_part)];
```

Comment:
A nice thing with the Lexicon is that it takes the enum type itself without a need to cast to an int-type.